geometry_msgs/Point32[] pointCoordinates
---
geometry_msgs/Point32[] dockCoordinates
